test/acceptance/dashboard_test.clj

```diff
@@ -107,8 +102,10 @@
   (t/testing "When the current user has already made Haggadot and goes to their dashboard, the Haggadot should be displayed in order from most recent to least recent"
     (create-haggadot haggadot "user1")
     (doto driver
-      (c/home->dashboard))
-    (let [titles (haggadot-titles (all-haggadot))]
+      (c/home->dashboard)
+      #_#(e/wait 9))
+    (let [_ (e/wait 9)
```

Comment:
Perhaps you could wait for something to happen instead of hardcoding the number `9` ... 

Also, why inside the `let`?

Reply:
I'll make it so I wait for the Haggadot to appear before continuing on.
I had the wait inside the let because it did not seem to work when it was outside the let.